samples/nrf9160/aws_fota/src/main.c

```diff
@@ -488,13 +491,17 @@ static void aws_fota_cb_handler(struct aws_fota_event *fota_evt)
 		if (err) {
 			printk("Error turning off the LTE link\n");
 		}
+		link_offline = true;
```

Comment:
I suggest adding a `break` after line 492, as `link_offline` can be set to true now even if `lte_lc_offline` failed

Reply:
ah yes thanks.